### ISSUE TITLE
(wront-type-argument stringp ("daemon"))

### ISSUE THREAD

**Issue by XO39** [emacs-lsp/lsp-dart#171]
After last update, I'm getting this error:
```
(wrong-type-argument stringp ("daemon"))
```

Here is the debugger log:
```
Debugger entered--Lisp error: (wrong-type-argument stringp ("daemon"))
  make-process(:name "LSP Dart - Flutter daemon" :buffer #<buffer *LSP Dart - Flutter daemon*> :command ("/path/to/flutter/bin/flutter" ("daemon")))
  apply(make-process (:name "LSP Dart - Flutter daemon" :buffer #<buffer *LSP Dart - Flutter daemon*> :command ("/path/to/flutter/bin/flutter" ("daemon"))))
  start-process("LSP Dart - Flutter daemon" #<buffer *LSP Dart - Flutter daemon*> "/path/to/flutter/bin/flutter" ("daemon"))
  apply(start-process "LSP Dart - Flutter daemon" #<buffer *LSP Dart - Flutter daemon*> "/path/to/flutter/bin/flutter" ("daemon"))
  start-file-process("LSP Dart - Flutter daemon" #<buffer *LSP Dart - Flutter daemon*> "/path/to/flutter/bin/flutter" ("daemon"))
  apply(start-file-process "LSP Dart - Flutter daemon" #<buffer *LSP Dart - Flutter daemon*> "/path/to/flutter/bin/flutter" ("daemon"))
  comint-exec-1("LSP Dart - Flutter daemon" #<buffer *LSP Dart - Flutter daemon*> "/path/to/flutter/bin/flutter" (("daemon")))
  comint-exec(#<buffer *LSP Dart - Flutter daemon*> "LSP Dart - Flutter daemon" "/path/to/flutter/bin/flutter" nil (("daemon")))
  make-comint-in-buffer("LSP Dart - Flutter daemon" #<buffer *LSP Dart - Flutter daemon*> "/path/to/flutter/bin/flutter" nil ("daemon"))
  lsp-dart-flutter-daemon-start()
  lsp-dart--activate-features()
  lsp--text-document-did-open()
  lsp--open-in-workspace(#s(lsp--workspace :ewoc nil :server-capabilities #<hash-table equal 17/17 0x47cf40dd> :registered-server-capabilities nil :root "/path/to/project..."
...
```

Is it a bug or am I missing some settings somewhere? (Didn't see anything in the docs)

If it's a bug, i hope to have a fix soon as this one is a showstopper, I had to disable `lsp-mode` entirely for flutter projects.

**Comment by ericdallo** [comment]
Oh, my bad, I will do a quick fix

**Comment by XO39** [comment]
And when I try to close Emacs, it won't close, shows this:

**Comment by ericdallo** [comment]
It should be fixed on master, will do a 1.22.2 release
